addons/mail/security/mail_security.xml

```diff
@@ -65,4 +65,15 @@
             <field name="perm_unlink" eval="False"/>
         </record>
 
+        <record id="res_users_settings_rule_user" model="ir.rule">
```

Comment:
No "admin" rule allowing to update it if necessary ? Just asking.

Reply:
I haven't thought about that. It seems pretty useless in this case, those are minor settings
I'm not sure if it's a guideline to automatically add admin access?

Anyway if we don't do it (and I prefer not to considering we are short on time) I'm guessing if it's really necessary the admin could use the system user to change them?

Reply:
It is a standard rule in odoo, unless security / privacy, admin should have all rights (and access to views) in order to ease debug in case of issue. Otherwise you have to dive into postgresql if something is not working as expected...

That's not that much time consuming, a rule and some views.

Reply:
I've added them, let me know if it's good enough. For the form view I decided to not define it and keep the basic one, as we want to show all fields anyway and I don't want to bother with a livechat override.

Reply:
At least if we can have a menu entry and an action in technical, as we have it for all mail models, quite useful. Default views are ok for a fast merge.